services/catalog/src/simcore_service_catalog/services/director.py

```diff
@@ -156,10 +156,8 @@ async def setup_director(app: FastAPI) -> None:
         with log_context(
             _logger, logging.DEBUG, "Setup director at %s", f"{settings.base_url=}"
         ):
-
-            client = DirectorApi(base_url=settings.base_url, app=app)
-
             async for attempt in AsyncRetrying(**_director_startup_retry_policy):
+                client = DirectorApi(base_url=settings.base_url, app=app)
```

Comment:
This fixes having to restart the catalog all the time if a connection is lost and safed me some pain...

Reply:
ah that is very nice